testing/checkin/runner/mod.rs

```diff
@@ -97,6 +97,24 @@ pub fn create_runtime(
   additional_extensions: Vec<Extension>,
 ) -> (JsRuntime, WorkerHostSide) {
   let (worker, worker_host_side) = worker_create(parent);
+
+  /* TODO: remove, this is a hack
```

Comment:
same nextest deal here. unless we get groups landed, but this would still fail in configs without groups.